docker/Dockerfile.native

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:20.04
+ARG DEBIAN_FRONTEND=noninteractive
+
+COPY common.sh lib.sh /
+RUN /common.sh
+
+COPY cmake.sh /
+RUN /cmake.sh
+
+COPY xargo.sh /
+RUN /xargo.sh
+
+ARG TARGET_ARCH
+ARG QEMU_ARCH=$TARGET_ARCH
+
+COPY qemu.sh /
+RUN /qemu.sh $QEMU_ARCH softmmu
+
+COPY dropbear.sh /
+RUN /dropbear.sh
+
+COPY linux-image.sh /
+RUN /linux-image.sh $TARGET_ARCH
+
+COPY linux-runner /
+
+ENV CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_RUNNER="/linux-runner $TARGET_ARCH"
```

Comment:
this should use `TARGETPLATFORM`, `TARGETOS`, and/or `TARGETARCH`

Reply:
I was planning to, but then it needs to normalize those values, which is something similar to what we do in #952, since the values normalize differently than what we currently accept as values. But yes, it should be done and would simplify things a lot.